manim/mobject/mobject.py

```diff
@@ -696,14 +696,12 @@ def get_color(self):
 
     ##
 
-    def save_state(self, use_deepcopy=False):
+    def save_state(self):
         if hasattr(self, "saved_state"):
             # Prevent exponential growth of data
             self.saved_state = None
-        if use_deepcopy:
-            self.saved_state = self.deepcopy()
-        else:
-            self.saved_state = self.copy()
+        self.saved_state = self.copy()
```

Comment:
**Question/issue :** Are you sure about removing `deepcopy` ? Why ? Mobjects are often nessted, so `deepcopy` can sometime be mandatory, no ?

Reply:
Mobject.copy now always uses deepcopy https://github.com/ManimCommunity/manim/blob/master/manim/mobject/mobject.py#L204

Reply:
My bad. Thank you !